packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -563,46 +557,6 @@ export function TokenSearch({
     }
 
     try {
-      // Native USDC on L2 won't have a corresponding L1 address
```

Comment:
Moved to `useSelectedToken` - now USDC address can be simply passed over and the logic is abstracted away in the custom hook